src/html-conventions-lib/data-type-html-conventions.xsl

```diff
@@ -137,7 +142,7 @@
     </xsl:template>
     
     
-    <xd:doc>
+<!--    <xd:doc>
```

Comment:
as seen in the changes in this pull request this was replaced by the new template elementUniqueName. It is commented out and not deleted just in case someone will want to revert the changes made in this pull request